incubator/logstash/templates/deployment.yaml

```diff
@@ -27,6 +27,8 @@ spec:
     {{- end }}
           livenessProbe:
             tcpSocket:
+              initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+              periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
```

Comment:
`initialDelaySeconds` and `periodSeconds` are properties of `livenessProbe` not `tcpSocket`

should be:
```
          livenessProbe:
            tcpSocket:
              port: {{ .Values.service.internalPort }}
            initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
            periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
```

Reply:
before the change:

```
~/.../charts/incubator on remotes/simt2/logstash
$ helm upgrade test-logstash ./logstash
Error: UPGRADE FAILED: error validating "": error validating data: [found invalid field periodSeconds for v1.TCPSocketAction, found invalid field initialDelaySeconds for v1.TCPSocketAction]
~/.../charts/incubator on remotes/simt2/logstash
$ helm upgrade test-logstash ./logstash
Error: UPGRADE FAILED: error validating "": error validating data: [found invalid field initialDelaySeconds for v1.TCPSocketAction, found invalid field periodSeconds for v1.TCPSocketAction]
```